quickwit/rest-api-tests/scenarii/field_capabilities/0001-field-capabilities.yaml

```diff
@@ -0,0 +1,224 @@
+# Test date _field_caps API
+method: [GET]
+engines:
+  - quickwit
```

Comment:
how far are we from elastic's api?

Reply:
The most important features are supported, some minor parameters are not implemented yet

Reply:
if we drop

```
engines:
- quickwit
```

which part breaks?

Reply:
it works without, seems to pick quickwit as default

Reply:
I mean running it against elasticsearch without it.

Reply:
The main differences are:
- Mixed types are not supported in elastic, so this test doesn't work in elastic. We get multiple entries for mixed types. https://github.com/quickwit-oss/quickwit/issues/3527#issuecomment-1831166121
- We have different coercion rules (e.g. i64, u64, f64 preference order)
- Mapping of text types. Currently we map text fields with fast to `keyword` and indexed to `text`. `keyword` is not on the same fieldname in elastic but a new one.
- We map our floats to `double`, but elastic seems to use `float` when the type is detected